homeassistant/components/logger.py

```diff
@@ -23,6 +27,22 @@
 LOGGER_DEFAULT = 'default'
 LOGGER_LOGS = 'logs'
 
+_LOGS_SCHEMA = vol.All(
+    cv.ensure_list,
+    [
+        vol.Schema({
+            cv.string: vol.In(vol.Lower(list(LOGSEVERITY))),
```

Comment:
`cv.string` is correct. From the `logger.getLogger(…)` [docs](https://docs.python.org/3.4/library/logging.html#logging.getLogger):

> Return a logger with the specified name or, if name is None, return a logger which is the root logger of the hierarchy. If specified, the name is typically a dot-separated hierarchical name like ‘a’, ‘a.b’ or ‘a.b.c.d’. Choice of these names is entirely up to the developer who is using logging.